docs/usersguide/r2s.rst

```diff
@@ -113,6 +113,96 @@ that can utilize these mesh-based sources is found in the PyNE user's guide entr
 Note that each of these source files must be renamed to "source.h5m" for this purpose.
 By using these sources for photon transport, the shutdown dose rate can be obtained. Tally results will have to be normalized by the total photon source intentity. This information is found in the "total_photon_source_intensites.txt" file printed out by r2s.py step2.
 
+****************
+PyNE R2S example
+****************
+
+Using a simple geometry as a example, here is how we perform R2S calculation.
+The example geometry composed of four cubes of size 10x10x10 cm3. Cubes (in green) at bottom left and up right are water. Bottom right cube (in red) is made of steel, and up left cube (in white) is void.
+There is a isotropic single energy (14 MeV) neutron point source in the middle of the bottom right cube.
+The cubes are irradiated by the source with a single 3.5 day pulse, the neutron intensity is 1e10 n/s.
+The following content demonstrates the process of caculate the shutdown dose rate of these cubes at the time of 1 hour after shutdown.
+Example files could be found in `r2s_example <https://github.com/pyne/data/blob/master/r2s_example.tar.gz>`_.
+>`_.
+
+
+.. figure:: r2s_example_geometry.png
+    :align: center
+
+    **Figure 1:** *Geometry of R2S example (X-Y cross section).*
+
+Build the model using trelis as following figure. Set material group and export to "geom_without_material.h5m".
+
+.. figure:: r2s_example_trelis_geometry.png
+    :align: center
+
+    **Figure 1:** *Trelis model for the example.*
+
+
+Prepare material library, build material library. The material lib is then generated in file "example_material_lib.h5".
+
+.. code-block:: bash
+
+   $ python make_example_material.py
+
+Combine the geometry file and the material library, using  the following command:
+
+.. code-block:: bash
+
+   $ cp geom_without_material.h5m geom.h5m
+
+   $ uwuw_preproc geom.h5m -v -l example_material_lib.h5
+
+Prepare input file, define source, tally and other data cards. Example input file could be seen in r2s_example/neutron_transport/input.
```

Comment:
Link to this file?

Reply:
All the example files are in a `r2s_example.tar.gz`, there isn't a link for this input file.